tools/docker/Makefile

```diff
@@ -131,7 +131,8 @@ KERNEL_BUILD_VERSIONS := 4.14.254 \
 	5.4.235 \
 	5.10.173 \
 	5.15.101 \
-	6.1.18
+	6.1.18 \
```

Comment:
I think we try to keep these on the newest patch versions. Consider bumping to `5.4.254`, `5.10.224`, `5.15.165`, `6.1.106`